File: core/src/main/java/io/outright/xj/core/application/Application.java
// Copyright (c) 2016, Outright Mental Inc. (http://outright.io) All Rights Reserved.
package io.outright.xj.core.application;

import java.io.IOException;

public interface Application {
  /**
   * Configure Application Server
   */
  void Configure(String... packages);

  /**
   * Start Application Server
   */
  void Start() throws IOException;

  /**
   * Stop Application Server
   */
  void Stop();

  /**
   * Base URI of Application Server
   * @return String
   */
  String BaseURI();
}

File: ui/player-web/src/js/application.js
// Copyright (c) 2018, Outright Mental Inc. (https://outrightmental.com) All Rights Reserved.

import {Player} from "player";

/**
 Application

 */
export class Application {

    /**
     * @type {Player}
     */
    player;

    /**
     * Create new application
     */
    constructor() {
        this.player = new Player();
    }

}
